spec-main/index.js

```diff
@@ -16,6 +16,12 @@ process.env.ELECTRON_DISABLE_SECURITY_WARNINGS = 'true';
 
 const { app, protocol } = require('electron');
 
+try {
+  require('fs').rmdirSync(app.getPath('userData'), { recursive: true });
```

Comment:
Is this needed ? the domain before `ready` event of an app is an undefined area, its hard to get the timing right to delete the user dir unless we emit some sort of signal from the native side. There can be some initialization on the native side that might write into the user dir early on, one example would be spellchecker. The newly failing linux specs are definitely related to this change.

Reply:
If there is need to ensure deletion of user data dir, I think its better to have `kUserDataDir` implemented https://source.chromium.org/chromium/chromium/src/+/master:chrome/common/chrome_switches.cc;l=594 and the ci test script launches the app explicitly with `--user-data-dir <path>` and deletes path after the test stage finishes.

Reply:
@deepak1556 I don't think its needed anymore.  I'll remove it. I figured out a way to just delete on WOA after all the test runs are done: https://github.com/electron/electron/pull/25018/commits/5af1fbd36fc4626be099dd442bef5236bafac980